### ISSUE TITLE
TypeError: Cannot read properties of undefined (reading 'documentClass')

### ISSUE THREAD

**Issue by cotakardel** [fantasycalendar/FoundryVTT-TokenEase#15]
Hello! 

I tried to change the default token settings on "token overrides" and it would not let me submit it and this error appears on the console:

foundry.js:5836 
TypeError: Cannot read properties of undefined (reading 'documentClass')
    at ClientDatabaseBackend._preUpdateDocumentArray (foundry.js:12586)
    at ClientDatabaseBackend._updateDocuments (foundry.js:12533)
    at ClientDatabaseBackend.update (commons.js:6320)
    at async Function.updateDocuments (commons.js:5674)
    at async TokenDocumentT20.update (commons.js:5771)

I managed to change it only on tokens that were already on scenes, and once I do I could not modify it anymore in any way.

I'm a noob, but tried to re-install everything, am I doing something wrong?

**Comment by Haxxer** [comment]
Could you please post the full versions of Foundry, the system you're playing, and the modules you have installed. If anything, try disabling other modules to make sure it's not Token-Ease.

**Comment by cotakardel** [comment]
I got it working!

I was changing configurations only on: Configure Settings>DefaultTokenSettings>TokenEase (Although I still could not change anything here)

In other hand on the Settings>TokenEase it functions normally without any errors detected. (I completely missed the configuration page)

Foundry version: https://foundryvtt.com/releases/10.285
System, Tomernta20 v1.4.006: https://foundryvtt.com/packages/tormenta20
![modules](https://user-images.githubusercontent.com/56280987/193480944-92b058f3-2b45-4e70-8953-03e0a7e4654c.png)

I tested Token-Ease without any modules on, and got the same results.

But I think overall it's solved ^^ Thanks!
